here/traffic/sql/trigger_here_insert.sql

```diff
@@ -0,0 +1,22 @@
+CREATE OR REPLACE FUNCTION here.here_insert_trigger()
+    RETURNS trigger 
+    LANGUAGE 'plpgsql'
+    COST 100
+    VOLATILE NOT LEAKPROOF SECURITY DEFINER
+AS $BODY$
+BEGIN 
+
+INSERT INTO here.ta VALUES (NEW.link_dir, 
+							NEW.tx::date,
+							NEW.tx::time without time zone,
+							NEW.tx,
+							NEW.length,
+							NEW.mean,
+							NEW.stddev,
+							NEW.min_spd,
+							NEW.max_spd,
+							NEW.confidence,
+                            NEW.sample_size); -- postgresql <=10 does not support ON CONFLICT DO NOTHING
```

Comment:
Can you link to this?

Reply:
Under 5.10.2.3. Limitations
> Using the ON CONFLICT clause with partitioned tables will cause an error, because unique or exclusion constraints can only be created on individual partitions. There is no support for enforcing uniqueness (or an exclusion constraint) across an entire partitioning hierarchy.

I'm guessing we will have to create additional triggers for each partitioned table if `ON CONFLICT` is needed.

Reply:
ah shoot well I guess at least insert will fail if trying to insert duplicate data, which is fine. It's not like other pipelines where we might get partial data